deploy/scripts/src/accounts/test.ts

```diff
@@ -174,10 +174,20 @@ export function changeEmail(): void {
     res = timeGroup(groups[1].split('::')[1], () => http.get(env.envURL, { redirects: 0 }), { isStatusCode302 })
 
     //02_OIDCStubCall
-    res = timeGroup(groups[2].split('::')[1], () => http.get(res.headers.Location), {
-      isStatusCode200,
-      ...pageContentCheck('API Simulation Tool')
-    })
+    res = timeGroup(
+      groups[2].split('::')[1],
+      () => {
+        const r = http.get(res.headers.Location)
+        if (!pageContentCheck('--API Simulation Tool--').validatePageContent(r)) {
+          console.log(r.html('h1').first().text())
```

Comment:
```suggestion
          console.log(groups[2].split('::')[1], r.html('h1').first().text())
```
Suggest adding some info to the start of the console logs, so that we can tell what the expected result is meant to be

Reply:
Or 
```suggestion
          console.log('Expected "API Simulation Tool", got: ',r.html('h1').first().text())
```